### DISCUSSION TITLE
Home Assistant integration - Naming

### DISCUSSION THREAD

**Discussion by JiriBalcar** [Koenkk/zigbee2mqtt#18506] [General]
Hi all,
I just installed new version of HomeAssistant (2023.8.0) and i received two warnings in Repairs section.

<img width="602" alt="Screenshot 2023-08-03 at 7 58 24" src="https://github.com/Koenkk/zigbee2mqtt/assets/6883250/c983aac1-0b5a-4272-8d48-4f2934031e96">

**Discovered entities with a name that starts with the device name**

- Some MQTT entities have an entity name that starts with the device name. This is not expected. To avoid a duplicate name the device name prefix is stripped of the entity name as a work-a-round. Please inform the maintainer of the software application that supplies the affected entities to fix this issue.

<img width="602" alt="Screenshot 2023-08-03 at 8 02 18" src="https://github.com/Koenkk/zigbee2mqtt/assets/6883250/d94af8eb-3cf3-4a47-8086-136351c1b15d">

**Discovered MQTT entities with a name that is equal to the device name**

- Some MQTT entities have an entity name equal to the device name. This is not expected. The entity name is set to null as a work-a-round to avoid a duplicate name. Please inform the maintainer of the software application that supplies the affected entities to fix this issue.

I would like to ask if this is something I should fix or it will be fixed in zigbee2mqtt?

Thanks

**Comment by Koenkk** [comment]
Duplicate of https://github.com/Koenkk/zigbee2mqtt/issues/18445
